specification/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -6,10 +6,20 @@ This is the AutoRest configuration file the Cognitive Services Language SDK.
 
 ## Releases
 
-The current preview release of Conversational Language Understanding and Text Analytics is 2022-03-01-preview.
+The current preview release of Conversational Language Understanding and Text Analytics is 2022-05-01-preview.
 
 The current stable release of Question Answering is 2021-10-01.
 
+```yaml
+tag: release_2022_05_01_preview
+add-credentials: true
+clear-output-folder: true
+openapi-type: data-plane
+directive:
+  - suppress: LongRunningResponseStatusCode
+    reason: The validation tools do not properly recognize 202 as a supported response code.
+```
+
```

Comment:
Do not duplicate the top-level arguments. Just update the existing `tag` to the new value and make sure there is only one of these blocks or the readme.md is invalid and tools (validation, code gen, etc.) will fail.